definitions/variable/energy/energy-final.yaml

```diff
@@ -467,3 +497,43 @@
       energy.
     sector: all
     unit:
+
+- Final Energy|Other:
+    description: Final energy consumption of other energy sources that do not fit to any
+      other category
+    unit: EJ/yr
+
+- Final Energy|Transportation|Bunkers:
```

Comment:
In a parallel discussion with @robertpietzcker and others in the NAVIGATE project, we agreed to move "Bunkers" to the same level as "Transportation".

Reply:
In Final energy this is clear, however I would like to clarify how to handle this in the Emissions variables.

I would suggest reporting bunkers in the root of the emissions categories (option 1. Emissions|CO2|Bunkers), instead of the same level as transportation (option 2. Emissions|CO2|Energy|Demand|Bunkers).

In that way, all energy aggregations are by default provided without bunkers, which is much more useful in national results, and bunkers can be sum up or not if you want to include them in the analysis. 
Option 1 would also allow direct comparison with UNFCCC/IPCC tables as far as I know.  

Did you discussed that also, or reached a decision about it?

Reply:
If I understand the notes from the meeting with @robertpietzcker and @gunnar-pik correctly, their proposal was to include CO2 emissions from bunkers as `Emissions|CO2|Energy|Demand|Bunkers`, not as you suggest like `Emissions|CO2|Bunkers`... Any comments?

Reply:
Agreed - it should be  `Emissions|CO2|Energy|Demand|Bunkers`, so that it's in the same level as transportation

Reply:
Thks. I changed a variable and a description in [this](https://github.com/openENTRANCE/openentrance/pull/137/commits/6c77a37c49d59ec07d30f866916ca7e70df0ca1f) commit to be compatible with this bunker emissions structure.

The final energy bunker variables are also compatible with what was discussed here ([commit](https://github.com/openENTRANCE/openentrance/pull/137/commits/6869bb47e6f0b95788bc57e574e7da846f7ba4df)).

I believe the pull request in its current state is compatible with all comments made so far here, please correct me if I am wrong (except for the tag system use that still needs to be implemented).
@danielhuppmann Would you agree on merging it so we can have a working framework for WP1 ECEMF results submission? Meanwhile I will work in parallel to submit a new pull request that makes use of the tag system whenever possible.